db/system_keyspace.hh

```diff
@@ -249,6 +251,9 @@ public:
 
     future<> remove_endpoint(gms::inet_address ep);
 
+    future<> quarantine_host(locator::host_id host_id);
```

Comment:
I think the names should maintain symmetry. quarantine_host() should perhaps be called store_quarantine_host(). I presume there will be a map of quarantined hosts outside system keyspace, which will be used as the source of truth.

Reply:
Makes sense.
There is a set of quarantined hosts kept in gossip, introduced later in this series.